docs/usage/dto/1-dto-factory.rst

```diff
@@ -0,0 +1,119 @@
+DTO Factory
+===========
+
+Litestar maintains a suite of DTO factory types that can be used to create DTOs for use with popular data modelling
+libraries, such as ORMs. These take a model type as a generic type argument, and create subtypes of
+:class:`AbstractDTOFactory <litestar.dto.factory.abc.AbstractDTOFactory>` that support conversion of that model type to
+and from raw bytes.
+
+The following factories are currently available:
+
+- :class:`DataclassDTO <litestar.dto.factory.stdlib.DataclassDTO>`
+- :class:`SQLAlchemyDTO <litestar.contrib.sqlalchemy.dto.SQLAlchemyDTO>`
+
+Using DTO Factories
+-------------------
+
+DTO factories are used to create DTOs for use with a particular data modelling library. The following example creates
+a DTO for use with a SQLAlchemy model:
```

Comment:
> **a** SQLAlchemy model:

Now I know how you pronounce SQL :grimacing:

Reply:
My my, this is controversial - "Sequel-Alchemy" or "Ess-Que-Ell-Alchemy".. I don't know if the community is ready to have this discussion.